docs/asciidoc/modules/ROOT/partials/usage/apoc.load.directory.async.adoc

```diff
@@ -187,6 +189,30 @@ We obtain this result calling `MATCH (n:JsonToNode) RETURN properties(n) as prop
 }
 |===
 
+
+=== List and remove procedures
+
+We can see the list of 
+
+[source,cypher]
+----
+CALL apoc.trigger.list();
```

Comment:
should this be trigger.list ?
where is the remove procedure?

Reply:
Changed with the correct one